tests/storage/utils.py

```diff
@@ -398,17 +399,16 @@ def create_cirros_dv(
     access_modes=None,
     volume_mode=None,
     client=None,
-    dv_size=Images.Cirros.DEFAULT_DV_SIZE,
 ):
     with create_dv(
         dv_name=f"dv-{name}",
         namespace=namespace,
         url=get_http_image_url(image_directory=Images.Cirros.DIR, image_name=Images.Cirros.QCOW2_IMG),
-        size=dv_size,
         storage_class=storage_class,
         access_modes=access_modes,
         volume_mode=volume_mode,
         client=client,
+        size=get_default_storage_profile_minimum_supported_pvc_size(),
     ) as dv:
```

Comment:
This function gets a `storage_class`, will `get_default_storage_profile_minimum_supported_pvc_size` return the value for the same  `storage_class`?